README.md

```diff
@@ -28,14 +28,15 @@ For the complete list of tests, check our
 [CI pipeline](https://buildkite.com/rust-vmm/kvm-ioctls-ci).
 
 Each individual test runs in a container. To reproduce a test locally, you can
-use the dev-container on both x86 and arm64.
+use the dev-container on both x86, arm64 and riscv64.
 
 ```bash
+# As of riscv64, the tag is v47-riscv
```

Comment:
How about "for running riscv64 tests, replace `v47` with `v47-riscv`. This provides an emulated riscv64 environment on a x86_64 host."

Reply:
Ah, yes, I'm not a native speaker, glad to have you to take care of this :)